![alt text](./pageheader_rose2_babies.jpg)

# Data Science in Medicine using Python

### Author: Dr Gusztav Belteki

# Test Notebook


In this Notebook we will test if you have set up your computer for the course correctly.

Please highlight the next field by clicking with your mouse outside the box, for example just under `In`. A frame should appear around the box and its colour should be gray. 

(If you clicked inside the box, the frame is green. Click outside or press `esc`.)

Now press `Shift + Enter`

In [1]:
import sys
import numpy as np
import pandas as pd
import matplotlib as mpl
import scipy as sc
import seaborn as sbs
import sklearn as sklearn
import nltk

For a couple second an asterisk will be shown while importing these packages is on the way. 

Press Shift + Enter again

In [2]:
print(f'Python version: {sys.version}')
print(f'pandas version: {pd.__version__}')
print(f'matplotlib version: {mpl.__version__}')
print(f'numPy version: {np.__version__}')
print(f'seaborn version: {sbs.__version__}')
print(f'seaborn version: {sbs.__version__}')
print(f'scikit-learn version: {sklearn.__version__}')
print(f'nltk version: {nltk.__version__}')

Python version: 3.7.9 (default, Aug 31 2020, 07:22:35) 
[Clang 10.0.0 ]
pandas version: 1.1.3
matplotlib version: 3.3.2
numPy version: 1.19.2
seaborn version: 0.11.0
seaborn version: 0.11.0
scikit-learn version: 0.23.2
nltk version: 3.5


The output should be something like this. 

`Python version: 3.7.9 (default, Aug 31 2020, 07:22:35) 
[Clang 10.0.0 ]
pandas version: 1.1.3
matplotlib version: 3.3.2
numPy version: 1.19.2
seaborn version: 0.11.0
seaborn version: 0.11.0
scikit-learn version: 0.23.2
nltk version: 3.5`

If you get this output you are all set for the course.

Your Python version should be 3.7 or higher.